.gitlab-ci.yml

```diff
@@ -605,6 +605,12 @@ API:
           - aws/rhel-10.0-ga-aarch64
         INTERNAL_NETWORK: ["true"]
 
+      # single test config for experimental feature
```

Comment:
(super nitpick) I wonder if this and the previous one (or even also the previous-previous one) commits could be combined but very subjective and fine as is.

Reply:
Done.